pkg/cmd/projectgroup/create/create.go

```diff
@@ -153,3 +128,19 @@ func PromptMissing(opts *CreateOptions) error {
 
 	return nil
 }
+
+func AskName(ask question.Asker, messagePrefix string, resourceDescription string, value *string) error {
```

Comment:
we talked about having this in a shared package, will that be in a separate PR?

Reply:
My goal here we to get everything ready @benPearce1 so he could write tests and finish off project create, as a result of forgot to move it 😅 . I will move it, and fix the small nits but there will be another PR after that updates all the commands to use the common shared questions.